services/frontend-service/src/assets/_base.scss

```diff
@@ -65,14 +65,14 @@ body,
 .text-regular {
     @extend .font;
     font-weight: 500;
-    font-size: 10px;
+    font-size: 12px;
```

Comment:
These scss classes dontt seem to be aligned with the Figma design
Is the Figma design outdated?

Reply:
Actually yes, it's outdated.
In Figma we still use 18px, but that was way too large, so we went with 10px... which is too small, so we're now at 12px.

Reply:
Okay, thank you!